src/ci/docker/host-x86_64/dist-x86_64-linux/Dockerfile

```diff
@@ -82,6 +82,10 @@ COPY host-x86_64/dist-x86_64-linux/build-clang.sh /tmp/
 RUN ./build-clang.sh
 ENV CC=clang CXX=clang++
 
+ENV PERF_COMMIT 1e19fc4c6168d2f7596e512f42f358f245d8f09d
+RUN curl -LS -o perf.zip https://github.com/rust-lang/rustc-perf/archive/$PERF_COMMIT.zip
+RUN unzip perf.zip && mv rustc-perf-$PERF_COMMIT rustc-perf
```

Comment:
You may want to do this in a single `RUN` command removing `perf.zip` afterwards to reduce the image size. Docker creates a new layer for each `RUN` command AFAIK.